### ISSUE TITLE
Scaling of arrows in quiver3-plots

### ISSUE THREAD

**Issue by MKure** [matlab2tikz/matlab2tikz#689]
When making a quiver3 plot in Matlab and exporting them using matlab2tikz, I noticed that the "scale" option set to 0 did not have any effect. Scale set to 0 should disable automatic scaling. 

From the [documentation](http://se.mathworks.com/help/matlab/ref/quiver3.html?refresh=true):
> `quiver3(...,scale)` automatically scales the vectors to prevent them from overlapping, and then multiplies them by scale. scale = 2 doubles their relative length, and scale = 0.5 halves them. Use scale = 0 to plot the vectors without the automatic scaling.

I guess I can use scale = 1 if my vectors don't overlap

KInd regards
Mathias

**Comment by egeerardyn** [comment]
I will have a look at it tonight. You might also want to check out #683 since we are in the process of revamping the quiver plots: maybe in the new approach this is no longer an issue (I would be surprised, though).

**Comment by egeerardyn** [comment]
I can confirm this behavior. I have patched it up for inclusion in the next release in #690.
The patch is rather simple, so if you're not familiar with how to get it from GitHub, you can manually patch your version by replacing
```matlab
    if getOrDefault(h, 'AutoScale', true)
        scaleFactor = getOrDefault(h,'AutoScaleFactor', 0.9) / maxLength;
    else
        scaleFactor = 1;
    end
```
with
```matlab
    if strcmpi((getOrDefault(h, 'AutoScale', 'on'), 'on')
        scaleFactor = getOrDefault(h,'AutoScaleFactor', 0.9) / maxLength;
    else
        scaleFactor = 1;
    end
```

**Comment by egeerardyn** [comment]
Fixed in #690.
